packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -207,6 +208,14 @@ module.exports = {
     ]
   },
   plugins: [
+    // Lint our CSS
+    new StyleLintPlugin({
+      // @remove-on-eject-begin
+      configFile: path.join(__dirname, '../.stylelintrc'),
+      // @remove-on-eject-end
+      context: paths.appSrc,
+      files: ['**/*.css'],
```

Comment:
Ah, nice idea. If @gaearon agrees we want to go this direction, I will make this change.